microsoft/testsuites/network/common.py

```diff
@@ -8,6 +8,7 @@
 from lisa import Environment, Node, RemoteNode, constants
 from lisa.features import NetworkInterface
 from lisa.nic import NicInfo, Nics
+from lisa.sut_orchestrator.azure.common import is_mana_device, is_vf_exist
```

Comment:
The test case shouldn't refer to Azure package, unless it's a Azure only test case.

Reply:
With the new changes, we should consider below,
1. Wrap all logic in the network feature. so Azure specified things in Azure, and abstract from standard interface. Like `is_sriov_enabled` check if the sriov enabled e2e, `get_vf_driver` to get the mlx or mana.
2. To be clearer on different concepts, what's difference among them, which are azure only, which are general concepts, like SRIOV, VF, Synthetic...

Reply:
NetworkInterface is for azure vm network interface, out of the VM, is it suitable to put the logic of VM there?

Reply:
Using the feature is one of the ways to wrap platform specific logic. But the nic can do similar things.